sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/synctask/history/HistoryDataSyncTask.java

```diff
@@ -108,6 +108,7 @@ private void getEstimatedRows() {
     }
     
     private void instanceSyncRunner() {
+        syncConfiguration.getReaderConfiguration().setTableNameMap(syncConfiguration.getTableNameMap());
```

Comment:
I think we can move the logic out to Controller.

Reply:
I will refactor configuration code.